quickff/settings.py

```diff
@@ -106,7 +106,8 @@ def is_existing_file_name(key, value):
     'log_level'             : [is_not_none, is_string, has_value(['silent','low','medium','high','highest'])],
     'log_file'              : [is_string, is_nonexisting_file_name],
     'program_mode'          : [is_not_none, has_value(['DeriveFF','MakeTrajectories','PlotTrajectories'])],
-    'only_traj'             : [is_not_none, is_string],
+    'only_traj'             : [is_string],
+    'dont_terms'            : [is_string],
```

Comment:
As suggested in my earlier comment, I would name it dont_traj instead of dont_terms